lib/rules/no-implicit-coercion.js

```diff
@@ -265,11 +275,11 @@ module.exports = {
                 const operatorAllowed = options.allow.indexOf("+") >= 0;
 
                 if (!operatorAllowed && options.string && isAppendEmptyString(node)) {
-                    context.report(
-                        node,
-                        "use `{{code}} = String({{code}})` instead.", {
-                            code: sourceCode.getText(getNonEmptyOperand(node))
-                        });
+                    const code = sourceCode.getText(getNonEmptyOperand(node, ""));
+
```

Comment:
`getNonEmptyOperand` only takes one argument (just `node`). Can we get rid of the second argument? Also, mind getting rid of this extra newline while you're at it?